iOS/docs/kickoff/configure-ci/README.md

```diff
@@ -0,0 +1,145 @@
+CI configuration for iOS projects
+-----------------
+
+This is a tutorial explaining how to configure continuous integration for iOS projects from scratch. Some steps can't be followed without being granted access to certain services: request them to an iOS' technical leader:
+
+* [Pablo Giorgi](mailto:pablo.giorgi@wolox.com.ar)
+* [Guido Marucci Blas](mailto:guidomb@wolox.com.ar)
+
+The following steps will guide you to setup a new project in [Bitrise](https://bitrise.io). Note that this guide is intended for standard projects, so if any custom configuration is needed some steps may change.
+
+## Check repo files...
+
+### Check the repository’s `bitrise.yml`. 
+
+It can be found [here](https://github.com/Wolox/ios-base-project/blob/master/bitrise.yml) in the `ios-base-project` root directory.
+
+Replace `REPO_SLUG`, `BITRISE_PROJECT_PATH` and `BITRISE_SCHEME` values with your app’s values.
+Keep in mind that the bitrise steps versions may be outdated, so check the [last versions](https://github.com/bitrise-io/bitrise-steplib/tree/master/steps) inside each steps folder and update the file accordingly.
+
+Our bitrise scripts will use [Fastlane](https://github.com/fastlane/fastlane), this gem is already added in the [Gemfile](https://github.com/Wolox/ios-base-project/blob/master/Gemfile). You can run `fastlane init` to initialize Fastlane with a basic configuration.
+
+## Check configuration files encryption...
+
+This step is optional and only useful if you need to encrypt files to be used in Bitrise. The provided `bitrise.yml` is not configured for encryption, so in case your project requires it follow these steps:
+
+### Encryption
+
+If you need to use any secret configuration keys in the Bitrise build, encrypt the xcconfig files with those keys. Check that the encrypted xcconfig file is the one used by the test target.
+
+```
+# Put all the files in a tar file
+tar cvf secrets.tar file1 file2 
+
+# Encrypt the tar file
```

Comment:
It should be in the same place than we add the others. I just added a link to the right place to check where they should be.